crates/service/src/worker.rs

```diff
@@ -30,16 +31,13 @@ impl Worker {
         });
         let protect = WorkerProtect { startup, indexes };
         sync_dir(&path);
-        self::metadata::Metadata::write(path.join("metadata"));
+        Version::write(path.join("metadata"));
```

Comment:
Not required for `VERSION` file?

Reply:
This line is removed in the next commit.